armotypes/usersnotificationsmethods.go

```diff
@@ -6,12 +6,12 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-func (ap NotificationParams) SetDriftPercentage(percentage int) {
+func (ap *NotificationParams) SetDriftPercentage(percentage int) {
 	ap.DriftPercentage = &percentage
 }
 
-func (ap NotificationParams) SetMinSeverity(severity int) {
-	ap.DriftPercentage = &severity
+func (ap *NotificationParams) SetMinSeverity(severity int) {
```

Comment:
we don't have severity type in armo api?

Reply:
no it is an just an int - same as in vulnerability struct